### PR TITLE
[asciidoc] add basic table cells support

### PR DESCRIPTION
**Pull request by jnavila**
For test and comment :+1: 

This PR introduces the management of table cells in asciidoc. The support for now is very basic and has some known limitations:

 * escaped  | are not managed. I couldn't come up with a correct regexp which would reject them. Maybe they can be managed later.
 * for cells that contain asciidoc block formatting ("a|" cells), the content is processed as a verbatim block. The general rule is that if a cell content spans on two lines or more, the text is treated as verbatim.

Both limitations may be released in the future. They are less prioritary because making complex stuff in tables is generally discouraged or thrown upon: this usually a smell that a table is used for formatting purpose and not for presentation of tabular data.

This behavior breaks the previous one, so it is introduced as an option `tablecells` of asciidoc plugin.

**TODO**: if accepted, 
 * add an entry in NEWS
 * document the option

### CODE REVIEW

File: lib/Locale/Po4a/AsciiDoc.pm [hunk 1]
```diff
@@ -128,6 +128,7 @@ sub initialize {
     $self->{options}{'macro'}='';
     $self->{options}{'style'}='';
     $self->{options}{'definitions'}='';
+    $self->{options}{'tablecells'}=0;
```
Comment (mquinson) [inline lib/Locale/Po4a/AsciiDoc.pm]: Could you please document this option at the top of the file?

Thanks at lot,

**Comment by mquinson** [comment]
The release is done, so I'm in good shape to include your PR once it's (rebased and) documented.

Thanks again,
Mt

**Comment by jnavila** [comment]
Now, it works correctly :tada:

**Comment by mquinson** [comment]
This is very good, many thanks. My only remark is that you could add a tiny bit more of comments in the hard part of your code (eg, near the huge split command followed by a variable capturing the result of a regex), but that's not too ugly either. 

Thanks again,
Mt

**Comment by jnavila** [comment]
I'd be glad to change this part: I couldn't find a simple way to split the lines into interlaced cell contents and cell separators. Please note, that there is no submatch in the regexp. For the remaining, it's quite straightforward.

**Pull request merged**